packages/calcite-components/src/components/tip/tip.scss

```diff
@@ -88,7 +86,7 @@
 ::slotted(a) {
   @apply focus-base;
 
-  color: var(--calcite-tip-slotted-link-text-color, var(--calcite-color-brand));
+  color: var(--calcite-color-brand);
```

Comment:
why did you remove `--calcite-tip-slotted-link-text-color`?

Reply:
IIRC, we added `--calcite-tip-slotted-link-text-color` initially in the component token update phase, but decided to not add component tokens for slotted elements in favor of having users override these directly.